google-cloud-speech/src/test/java/com/google/cloud/speech/v1/it/ITSpeechTest.java

```diff
@@ -76,7 +76,7 @@ public void longrunningRecognize() throws Exception {
   @Test
   public void streamingRecognize() throws Exception {
     byte[] audioBytes =
-        Resources.toByteArray(new URL("https://storage.googleapis.com/gapic-toolkit/hello.flac"));
+        ByteString.readFrom(new FileInputStream(RESOURCES + "hello.flac")).toByteArray();
```

Comment:
Can we use [`ClassLoader#getResourceAsStream()`](https://docs.oracle.com/javase/8/docs/api/java/lang/ClassLoader.html#getResourceAsStream-java.lang.String-) instead of using a relative path from the module root?

Something like `getClass().getClassLoader().getResourceAsStream("hello.flac")`